roles/node/molecule/default/molecule.yml

```diff
@@ -24,6 +24,8 @@ provisioner:
     D: True
 verifier:
   name: ansible
+  options:
+    D: True
```

Comment:
do you need this?

Reply:
It adds `--diff` for ansible. It's more informative for debugging.

Reply:
Cool I did not know about it, very useful!